Tests are order dependent

this combination fails

```
@RunWith(Suite.class)
@SuiteClasses({
		org.eclipse.xtext.enumrules.GrammarParserTest.class,
		org.eclipse.xtext.validation.CompositeValidatorWithEObjectValidatorTest.class,

})
public class Viech {

}

```

@szarnekow any idea?

```
java.lang.ExceptionInInitializerError
	at org.eclipse.xtext.validation.CompositeValidatorWithEObjectValidatorTest$1$1.configure(CompositeValidatorWithEObjectValidatorTest.java:38)
	at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:223)
	at com.google.inject.spi.Elements.getElements(Elements.java:101)
	at com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:133)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:103)
	at com.google.inject.Guice.createInjector(Guice.java:95)
	at com.google.inject.Guice.createInjector(Guice.java:72)
	at com.google.inject.Guice.createInjector(Guice.java:62)
	at org.eclipse.xtext.validation.CompositeValidatorWithEObjectValidatorTest$1.createInjector(CompositeValidatorWithEObjectValidatorTest.java:31)
	at org.eclipse.xtext.enumrules.EnumRulesTestLanguageStandaloneSetupGenerated.createInjectorAndDoEMFRegistration(EnumRulesTestLanguageStandaloneSetupGenerated.java:23)
	at org.eclipse.xtext.tests.AbstractXtextTests.with(AbstractXtextTests.java:107)
	at org.eclipse.xtext.validation.AbstractCompositeValidatorTest.setUp(AbstractCompositeValidatorTest.java:47)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: java.lang.NullPointerException
	at org.eclipse.xtext.enumrules.enumRulesTestLanguage.impl.EnumRulesTestLanguagePackageImpl.initializePackageContents(EnumRulesTestLanguagePackageImpl.java:226)
	at org.eclipse.xtext.enumrules.enumRulesTestLanguage.impl.EnumRulesTestLanguagePackageImpl.init(EnumRulesTestLanguagePackageImpl.java:98)
	at org.eclipse.xtext.enumrules.enumRulesTestLanguage.EnumRulesTestLanguagePackage.<clinit>(EnumRulesTestLanguagePackage.java:58)
	... 44 more

```

--- a/org.eclipse.xtext.tests/src/org/eclipse/xtext/validation/CompositeValidatorWithEObjectValidatorTest.java
+++ b/org.eclipse.xtext.tests/src/org/eclipse/xtext/validation/CompositeValidatorWithEObjectValidatorTest.java
@@ -10,9 +10,11 @@ package org.eclipse.xtext.validation;
 
 import java.util.Collection;
 
+import org.eclipse.emf.ecore.EPackage;
 import org.eclipse.emf.ecore.EValidator;
 import org.eclipse.xtext.enumrules.EnumRulesTestLanguageStandaloneSetup;
 import org.eclipse.xtext.enumrules.enumRulesTestLanguage.EnumRulesTestLanguagePackage;
+import org.eclipse.xtext.enumrules.enums.EnumsPackage;
 import org.eclipse.xtext.validation.CompositeEValidator.EValidatorEqualitySupport;
 import org.junit.Test;
 
@@ -28,6 +30,8 @@ public class CompositeValidatorWithEObjectValidatorTest extends AbstractComposit
 		return new EnumRulesTestLanguageStandaloneSetup() {
 			@Override
 			public Injector createInjector() {
+				EnumsPackage enumsPack = EnumsPackage.eINSTANCE;
+				EPackage.Registry.INSTANCE.put(enumsPack.getNsURI(), enumsPack);
 				return Guice.createInjector(new org.eclipse.xtext.enumrules.EnumRulesTestLanguageRuntimeModule() {
 					
 					@Override

--- a/org.eclipse.xtext.tests/src/org/eclipse/xtext/validation/CompositeValidatorWithoutEObjectValidatorTest.java
+++ b/org.eclipse.xtext.tests/src/org/eclipse/xtext/validation/CompositeValidatorWithoutEObjectValidatorTest.java
@@ -10,9 +10,11 @@ package org.eclipse.xtext.validation;
 
 import java.util.Collection;
 
+import org.eclipse.emf.ecore.EPackage;
 import org.eclipse.emf.ecore.EValidator;
 import org.eclipse.xtext.enumrules.EnumRulesTestLanguageStandaloneSetup;
 import org.eclipse.xtext.enumrules.enumRulesTestLanguage.EnumRulesTestLanguagePackage;
+import org.eclipse.xtext.enumrules.enums.EnumsPackage;
 import org.eclipse.xtext.validation.CompositeEValidator.EValidatorEqualitySupport;
 import org.junit.Test;
 
@@ -28,6 +30,8 @@ public class CompositeValidatorWithoutEObjectValidatorTest extends AbstractCompo
 		return new EnumRulesTestLanguageStandaloneSetup() {
 			@Override
 			public Injector createInjector() {
+				EnumsPackage enumsPack = EnumsPackage.eINSTANCE;
+				EPackage.Registry.INSTANCE.put(enumsPack.getNsURI(), enumsPack);
 				return Guice.createInjector(new org.eclipse.xtext.enumrules.EnumRulesTestLanguageRuntimeModule() {
 					
 					@Override

